src/api/v1/users/__tests__/index.test.integration.ts

```diff
@@ -1,23 +1,29 @@
 /*
  * User API functional tests
  */
+import * as Hapi from 'hapi';
 import * as Knex from 'knex';
 import routes from '..';
 import { init } from '../../../../../tests/server';
+import addHooks from '../../hooks';
 import { getConfig } from '../../../../../config';
 
 describe('API /users', () => {
+  let server: Hapi.Server;
   const config = getConfig(process.env.NODE_ENV);
   const knex = Knex(config.knex);
 
+  beforeEach(async () => {
+    server = await init(config, { routes, knex });
+    addHooks(server);
```

Comment:
shouldn't this be added in `tests/server.ts`?

Reply:
Fair

Reply:
Although I'd actually prefer to do this in #96 to avoid conflicts.